.release-please-manifest.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
We upgraded release please from v3 -> v4 in bson@6.8.  So backports earlier than that need to add the manifest and config files to work with release please v4.